.travis.yml

```diff
@@ -9,10 +9,10 @@ services:
   - mongodb
 
 rvm:
-  - 2.3.8
```

Comment:
Since Ruby 2.3 hasn't reached EOL just yet, and we haven't officially decided to drop support for it, I feel like we should continue to test against it for now.

Reply:
I think it did go EOL last year: https://www.ruby-lang.org/en/news/2019/03/31/support-of-ruby-2-3-has-ended/ ; but I agree with your sentiment and have added it back until a later date.

Reply:
You are 💯 correct. Apparently, I don't know what year it is currently.